SwiftKitCore/src/main/java/org/swift/swiftkit/core/SwiftInstance.java

```diff
@@ -17,49 +17,38 @@
 import java.util.concurrent.atomic.AtomicBoolean;
 
 public abstract class SwiftInstance {
-    /// Pointer to the "self".
-    private final long selfPointer;
```

Comment:
"The change" the `SwiftInstance` does not need storage, we can just have FFM or JNI one implement the abstract method instead